salt/master.py

```diff
@@ -1276,6 +1276,14 @@ def __verify_load(self, load, verify_keys):
 
         return load
 
+    def _file_envs(self, *args):
+        '''
+        Return all fileserver backends
+
+        :param dict load: A payload received from a minion (ignored)
+        '''
+        return self._fs_file_envs()
```

Comment:
Should this be a function call, or an attribute?

Reply:
It has historically been a function call but with the wrong arguments.